### ISSUE TITLE
last_api_update() ifelse logic problem if status != 200

### ISSUE THREAD

**Issue by jeffeaton** [OJWatson/rdhs#40]
If `updates$status_code == 200` is `FALSE`, then the ifelse does not assign a value to `date`, and so the function simply returns the `base::date()` function.

Somehow I'm getting a status_code of 500, not sure why, but it's preventing package load:

```
> library(rdhs)
Error: package or namespace load failed for ‘rdhs’:
 .onAttach failed in attachNamespace() for 'rdhs', details:
  call: `>.default`(last_api_update(), cache_date)
  error: comparison (6) is possible only for atomic and list types
```

I suspect this is a misplaced brace in the if-else logic.

Here's the current function:

```
last_api_update <- function() {

  if (Sys.getenv("rdhs_TIMEOUT") == "") {
    Sys.setenv("rdhs_TIMEOUT" = 30)
  }

  updates <- tryCatch(
    httr::GET(
      "https://api.dhsprogram.com/rest/dhs/dataupdates",
      httr::timeout(as.numeric(Sys.getenv("rdhs_TIMEOUT")))
    ),
    error = function(e) NULL
  )

  if (inherits(updates, "response")) {
    if (updates$status_code == 200) {
      updates <- rbind_list_base(handle_api_response(updates)$Data)
      date <- updates$UpdateDate %>%
        mdy_hms() %>%
        max()
    }
  } else {

    date <- -0.5
    message("The DHS API took longer than ", Sys.getenv("rdhs_TIMEOUT"),
            " seconds to respond, or it returned an error.\n",
            "As a result some of the functionality of rdhs may not work.\n",
            "To check if the API is down please head to:\n",
            "https://api.dhsprogram.com/rest/dhs/dataupdates")
  }

  date
}
```

**Comment by jeffeaton** [comment]
I've patch my version like this for now: 

```
...

  if (inherits(updates, "response") && updates$status_code == 200) {
    updates <- rbind_list_base(handle_api_response(updates)$Data)
    date <- updates$UpdateDate %>%
      mdy_hms() %>%
      max()
  } else {

    date <- -0.5
...
```

**Comment by jeffeaton** [comment]
Fixed by dbe0d5a
